app/main/helpers/validation.py

```diff
@@ -183,33 +183,35 @@ class DOSValidator(DeclarationValidator):
         "appropriateTradeRegisters", "appropriateTradeRegistersNumber",
         "licenceOrMemberRequired", "licenceOrMemberRequiredDetails",
     ])
+
+    dependent_fields = {
+        # If you responded yes to any of questions 22 to 34
+        "mitigatingFactors": [
+            'misleadingInformation', 'confidentialInformation', 'influencedContractingAuthority',
+            'witheldSupportingDocuments', 'seriousMisrepresentation', 'significantOrPersistentDeficiencies',
+            'distortedCompetition', 'conflictOfInterest', 'distortingCompetition', 'graveProfessionalMisconduct',
+            'bankrupt', 'environmentalSocialLabourLaw', 'taxEvasion'
+        ],
+        # If you responded yes to either 36 or 37
+        "mitigatingFactors2": [
+            "unspentTaxConvictions", "GAAR"
+        ],
+        # If you responded yes to 50 (Modern Slavery)
+        "modernSlaveryStatement": [
+            "modernSlaveryTurnover",
+        ],
+        "modernSlaveryReportingRequirements": [
+            "modernSlaveryTurnover"
+        ],
+    }
+
     email_validation_fields = set(["contactEmailContractNotice", "primaryContactEmail"])
     character_limit = 5000
 
     def get_required_fields(self):
```

Comment:
To flake8 ignore just this method as mentioned in previous comment:

```suggestion
    def get_required_fields(self):  #noqa: C901
```